cpu/BUILD

```diff
@@ -90,6 +96,11 @@ constraint_value(
     constraint_setting = ":cpu",
 )
 
+constraint_value(
```

Comment:
@katre Just putting an idea in your head.  Not part of the review.

Perhaps it is time for a `constants` namespace or maybe just a CPU properties one.  It would define bits32, bits64, bigendian, littleendian...  

Then... we change the documentation style here to hint that your platform constraints should use both ppc and bits32. Or maybe start defining some base platforms of CPU type and bits, which people are encouraged to inherit from.